clients/spark/core/src/main/scala/io/treeverse/clients/GarbageCollector.scala

```diff
@@ -381,18 +388,18 @@ object S3BulkDeleter {
         .collect()
   }
 
-  def main(args: Array[String]): Unit = {
-    if (args.length != 4) {
-      Console.err.println(
-        "Usage: ... <repo_name> <runID> <region> s3://storageNamespace/prepared_addresses_table"
-      )
-      System.exit(1)
-    }
-    val repo = args(0)
-    val runID = args(1)
-    val region = args(2)
-    val addressesDFLocation = args(3)
-    val spark = SparkSession.builder().getOrCreate()
-    remove(repo, addressesDFLocation, runID, region, spark)
-  }
+  // def main(args: Array[String]): Unit = {
```

Comment:
How does this Spark job run without the main function?

Reply:
There's another "main" function that gets used.  This used to be another execution method, it was never finished -- so I unified the two objects and got rid of this one.

Sorry, will delete the comment.